src/pages/cronLogs.tsx

```diff
@@ -0,0 +1,73 @@
+import type { NextPage } from "next";
+import { useSession } from "next-auth/react";
+import {
+  Text,
+  Table,
+  Thead,
+  Tr,
+  Th,
+  Tbody,
+  Td,
+  Card,
+  HStack,
+  VStack,
+  Progress,
+} from "@chakra-ui/react";
+import { trpc } from "utils/trpc";
+import { useEffect } from "react";
+import Router from "next/router";
+import AdminNav from "components/AdminNav";
+
+const CronLogs: NextPage = () => {
+  const { data: session, status } = useSession();
+  useEffect(() => {
+    if (!session) Router.push("/admin");
+  });
+
+  const cronlogs = trpc.cronlogs.list.useQuery(undefined, { enabled: !!session }).data?.entries;
+
+  if (session && status === "authenticated") {
+    if (!cronlogs)
+      return (
+        <>
+          <AdminNav />
```

Comment:
This is fine for now. But I think we should just have a single Admin NextPage, then all the admin pages now become components on that page ... I think that's a better way ... Somehow we need to have only a single use of AdminNav ... I'll make an issue